test/cql-pytest/run

```diff
@@ -27,7 +27,16 @@ run.wait_for_services(pid, [
     lambda: run.check_rest_api(ip),
     lambda: check_cql(ip)
 ])
-success = run.run_pytest(sys.path[0], ['--host=' + ip] + sys.argv[1:])
+
+args = ['--host=' + ip]
+if not any(map(lambda x: x.startswith('--replication'), sys.argv)):
+    # The default for --replication is "all". To faciliate iterative
+    # development, with short compile+test cycles, we only run the test with
+    # --replication=tablets, when running them via 'run'.
+    # If the user already provided this option, don't overwrite it.
+    args.append('--replication=tablets')
+
+success = run.run_pytest(sys.path[0], args + sys.argv[1:])
```

Comment:
Can we keep `run` running a single selection of tablets/vnodes, and make the duplication happen in test.py? Perhaps as two virtual suites?

I don't know why I'm asking it, I don't have a concrete reason, it's just how I would have done it. That's not enough to justify a change.

Reply:
This is what I've also been asking. Just like test.py knows how to run cql-pytest twice, once for debug build and once for release build, and we don't have code in cql-pytest (or test/cql or test/alternator or test/boost or ...) that knows it should run each test twice in two build modes.

Reply:
There is a very simple reason for building this switching into `run` + `conftest.py`: simplicity. I did this in less then 2 dozens of lines of code (half of which is comments).

If I were to do this in `test.py`, I'd had to write a dedicated test suite class + test class. The currently used PythonSuite + PythonTest are shared by other tests, so I can't just willy-nilly change it. Then I would have to make sure the test suites are distinguishable in the output, they match the command line filters, etc. This will be hundreds of lines of code and it would probably take me days to do.

> Just like test.py knows how to run cql-pytest twice, once for debug build and once for release build

This sounds simple but it is not. I would have to add a new global test parameter, like mode, which would result in changing **all** the existing test suites. Most test suites would not use this new parameter, but I would still have to make sure they don't choke on it. 

Maybe we'll have to do this anyway, when converting the other test suites. But I'm really not itching to do this before I know I must do it.